docs/maintain/maintain-wss.md

```diff
@@ -1,86 +1,110 @@
-Never open websockets to your validator node - there's no reason to do that and it can only lead to
-security gaffes._
-
-In this guide we'll be using Ubuntu 18.04 hosted on a \$10 DigitalOcean droplet. We'll assume you're
-using a similar OS, and that you have nginx installed (if not, run `sudo apt-get install nginx`).
+:::note Enabling remote access to your validator node should not be necessary and is not
+suggested as it can often lead to security problems :::
 
 ## Set up a node
 
-Whether it's a generic Substrate node, a Kusama node, or your own private blockchain, they all
-default to the same websocket connection: port 9944 on localhost. For this example, we'll set up a
-Kusama sync node (non-validator).
+Setting up any Substrate-based node relies on a similar process.  For example, they will all by default share the same websocket connection at port 9944 on localhost.
+In this example, we'll set up a Kusama sync node on a debian flavoured server (such as Ubuntu 22.04). Create a new server on your provider of choice or locally at home. See
+[Set up a Full Node](./maintain-sync) for additional instructions. You can choose to install from the default
+apt repository or build from scratch. The startup options  in the setup process provide a variety of 
+different settings that can be modified.
 
-Create a new server on your provider of choice or locally at home (preferred). We'll assume you're
-using Ubuntu 18.04. Then install Substrate and build the node.
+A typically setting for an externally accessible kusama archive RPC node would be:
 
-```bash
-curl https://getsubstrate.io -sSf | bash
-git clone https://github.com/paritytech/polkadot kusama
-cd kusama
-./scripts/init.sh
-cargo build --release
-./target/release/polkadot --name "DigitalOcean 10 USD droplet ftw" --rpc-cors all
+```config
+polkadot --chain kusama --name myrpc --state-pruning archive --blocks-pruning archive --ws-max-connections 100 --rpc-cors all --rpc-methods Safe --ws-port 9944
+```
+
+Or for a kusama pruned RPC node:
+
+```config
+polkadot --chain kusama --name myrpc --state-pruning 1000 --blocks-pruning archive --ws-max-connections 100 --rpc-cors all --rpc-methods Safe --ws-port 9944
 ```
 
-This will start the syncing process with Kusama's mainnet.
+The specified flag option are outlined in greater detail below.
+
+### Archive Node vs Pruned Node
+
+A pruned node only tracks more recent information about the network and not its full history. Most
+frequently required actions can be completed with a pruned node, such as displaying account balances, making
+transfers, setting up session keys, staking, etc. An archive node has the full history (database) of the
+network and can be queried in all kind of ways, such as providing historical information regarding transfers, balance histories, and more advanced queries involving past events, etc.
+
+An archive node requires a lot more diskspace ([overview](https://stakeworld.io/snapshot)). For an
```

Comment:
> [overview](https://stakeworld.io/snapshot)

Maybe let's make clear that it is an external link. For example, using a sentence like `According to statistics provided by Stakework, on Jan 6th 2023 a polkadot prune node was 100GB.`